go.mod

```diff
@@ -3,14 +3,9 @@ module github.com/awslabs/amazon-qldb-driver-go
 go 1.14
 
 require (
-	github.com/amzn/ion-go v1.1.3
```

Comment:
Why did we remove this?

Reply:
Those dependencies are not necessary for that particular `go.mod` file. But they do exist in `v2/go.mod`.

Reply:
Why are they not necessary?

Reply:
Turns out this entire file is unnecessary (and also responsible for breaking the CodeQL analysis) so it's removed.